docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,115 +1,135 @@
-= Contracts for Cairo
+:starknet: https://starkware.co/product/starknet/[Starknet]
+:scarb: https://docs.swmansion.com/scarb[Scarb]
+:installation: https://docs.swmansion.com/scarb/download.html[this guide]
 
-*A library for secure smart contract development* written in Cairo for https://starkware.co/product/starknet/[StarkNet], a decentralized ZK Rollup.
+= Contracts for Cairo
 
-== Usage
+*A library for secure smart contract development* written in Cairo for {starknet}, a decentralized ZK Rollup.
 
 WARNING: This repo contains highly experimental code. Expect rapid iteration. *Use at your own risk.*
 
-=== First time?
+== Overview
+
+=== Installation
 
-Before installing Cairo on your machine, you need to install `gmp`:
+The library is available as a {scarb} package. Follow {installation} for installing Cairo and Scarb on your machine
+before proceeding, and run the following command to check that the installation was successful:
 
 [,bash]
 ----
-sudo apt install -y libgmp3-dev # linux
-brew install gmp # mac
-----
+$ scarb --version
 
-TIP: If you have any trouble installing gmp on your Apple M1 computer, https://github.com/OpenZeppelin/nile/issues/22[here's a list of potential solutions].
+scarb 0.7.0 (58cc88efb 2023-08-23)
+cairo: 2.2.0 (https://crates.io/crates/cairo-lang-compiler/2.2.0)
+sierra: 1.3.0
+----
 
-=== Set up your project
+==== Set up your project
 
-Create a directory for your project, then `cd` into it and create a Python virtual environment.
+Create an empty directory, then `cd` into it, and initialize a new scarb project by running:
 
 [,bash]
 ----
-mkdir my-project
-cd my-project
-python3 -m venv env
-source env/bin/activate
+mkdir my_project && cd my_project
+scarb init
```

Comment:
i'd separate this in two code blocks, or add `$` at the beginning to be clear that these are different commands. separating the blocks would also make them copy-pasteable

Reply:
I rather the leading $ if you are ok with it. I don't think two blocks add that much benefit, and it looks broken in the UI imo.

Reply:
To me, having them on separate lines is enough to convey that they're different commands. In the current README (cairo v0), we have a few instances of a single code block hosting multiple commands. That doesn't mean it's best by any means, just pointing out that we did this in this past.

If, however, we insist on not keeping it as-is, what about adding an intro sentence between the blocks? e.g.

----

Create an empty directory and `cd` into it:
```
mkdir my_project && cd my_project
```
Initialize a new Scarb project:
```
scarb init
```

Reply:
@ericnordelo the added value i see for separation is easy copy+paste. i don't think it looks bad, and if you do then we can just add a sentence like andrew proposes.